Jenkinsfile

```diff
@@ -15,7 +15,7 @@ pipeline {
 		timestamps()
 
 		// Prevent concurrent builds
-		disableConcurrentBuilds();
+		disableConcurrentBuilds()
 	}
 	stages {
 		stage ('1.3.3 Build') {
```

Comment:
The version slightly further down here will need updating to the next version before this gets merged as per the readme for releasing